packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -303,6 +305,37 @@ function addDebugger(app) {
   }
 }
 
+function wrapCommands(app) {
+  const proto = Object.getPrototypeOf(app.client);
+  const commands = Object.keys(proto).filter((key) => {
```

Comment:
This is admittedly a hack. There are various other things we can do, most of them are hacks anyway. We could also just hardcode which ones we want to wrap? Either way I think the benefits are worth it.